data_structures/import_edge.hpp

```diff
@@ -73,12 +75,14 @@ struct EdgeBasedEdge
                            const NodeID target,
                            const NodeID edge_id,
                            const EdgeWeight weight,
+                           const EdgeWeight duration,
                            const bool forward,
                            const bool backward);
     NodeID source;
     NodeID target;
     NodeID edge_id;
-    EdgeWeight weight : 30;
+    EdgeWeight weight;
+    EdgeWeight duration : 30;
```

Comment:
Until now, the weight was the duration. The upper bounds are known and this won’t break things. However if someone has strange ideas concerning the weight, the are more chances that bigger values are obtained. That is why I kept the duration padded and not the weight.